### PR TITLE
Add Enhance Content Section to Help Center

### PR DESCRIPTION
**Pull request by jayk-u**
This new section uses content from the previous help center mainly:
- Markdown Support
- Topics

Both pages have been rewritten to make them easier to read, with some information corrected or clarified through context. Several typos and other corrections to the original content have been applied.

@thimy I took the liberty of adding a new icon to the sidebar. I'll let you be the judge of its visual relevance (I'm not always very good at that).

### CODE REVIEW

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/markdown-support.md:10]:
```suggestion
To allow you to enrich your documentation, we support common Markdown syntax, including call-outs and the addition of images, for example.
```
Keeping it positive all the way!

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/markdown-support.md:115]: How about writing an example for each supported case, so that everything is crystal clear? Maybe it's just me, but I'm now a little confused about how exactly I should write the "dimension" part. I'd recommend that the example shows the whole parenthesis, including the path, "image title", and so on.

File: src/_help/enhance-documentation-content/topics.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/topics.md:8]: Is this working both for OpenAPI and AsyncAPI?

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]

File: src/_help/enhance-documentation-content/topics.md
Comment (thimy) [reply src/_help/enhance-documentation-content/topics.md:8]: Yes it does 👍

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (thimy) [inline src/_help/enhance-documentation-content/markdown-support.md:80]: I would even say something among those lines: 
```suggestion
You should add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) when introducing an image, to make it accessible to everyone.
```
or 
```suggestion
Don't forget to add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, to make it accessible to everyone.
```
Reminding people it's optional might encourage them not to put them but they really really should have alts for every image.

File: src/_help/enhance-documentation-content/topics.md
Comment (thimy) [inline src/_help/enhance-documentation-content/topics.md:10]:
```suggestion
By default, most specification standards do not support the addition of extra content, so we added this additional property to insert at the beginning of your specification file: `x-topics`.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (jayk-u) [reply src/_help/enhance-documentation-content/markdown-support.md:115]: Don't we have that example on line 94? Or would you like something else?

File: src/_help/enhance-documentation-content/topics.md
Comment (jayk-u) [reply src/_help/enhance-documentation-content/topics.md:10]: Not 100% sure of the transition here and that would make a super long sentence.

Comment (thimy) [reply src/_help/enhance-documentation-content/topics.md]: That was just a suggestion because the rhythm of that part sounded a bit weird to me but you make the call. :)

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:8]:
```suggestion
The generation of your API documentation is based on your API document, which must adhere to one of the standards we support (OpenAPI/Swagger and AsyncAPI.).
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:12]:
```suggestion
You can integrate Markdown directly into your API document or as an [external reference](/help/enhance-documentation-content/markdown-support/#markdown-files-as-an-external-reference) pointing to Markdown files.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:140]:
```suggestion
Markdown files can be included as an [external reference](/help/specification-support/references) within your API document with the $ref syntax `$ref: "./path/to/local-markdown.md"`.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:142]:
```suggestion
In the same way you can extract part of your API definition (usually JSON schema of your models into dedicated `*.yaml` or `*.json` files), you can extract your markdown content into dedicated files too.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:144]:
```suggestion
**E.g.** Your OpenAPI file `api-contract.yml` can thus looks like:
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:168]:
```suggestion
These files will be rendered within your documentation as if they were part of your API document.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:170]: I don't get this line, I think we can remove it.

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:162]: Can we remove the `x-topics` part of the examples here, to keep it simple? Keeping only the $ref: "docs/introductions.md" seems to be enought.

File: src/_help/enhance-documentation-content/topics.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/topics.md:8]:
```suggestion
`x-topics` is a vendor-specific property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
```

File: src/_help/enhance-documentation-content/topics.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/topics.md:10]: I think we can remove this sentence following my previous proposal, as we directly mention the property. I find it more clear like this, WDYT?

File: src/_help/enhance-documentation-content/topics.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/topics.md:41]: Bad URL

**Review by scharrier — COMMENTED** [review]
Great job! I left some comments, please be careful to not use "API specification" anymore, or only to talk about the standard.

Also, you'll need to update the `_redirects` file to add redirections for the 2 moved files (as they URL will change).

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (jayk-u) [reply src/_help/enhance-documentation-content/markdown-support.md:168]: way much better

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (jayk-u) [reply src/_help/enhance-documentation-content/markdown-support.md:162]: I removed also the mention of topics at the end, as I wasn't able to find an appropriate way to introduce them here.

File: src/_help/enhance-documentation-content/topics.md
Comment (jayk-u) [reply src/_help/enhance-documentation-content/topics.md:8]: For my own curiosity: why "vendor-specific"? What is a vendor?

File: src/_help/enhance-documentation-content/index.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/index.md:8]:
```suggestion
API documentation deserves a bit more writing than a plain technical description. It can be useful to add an introduction or a quick tutorial. You might want to provide context, additional reading, or understanding materials.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/markdown-support.md:8]:
```suggestion
The generation of your API documentation is based on your API definition, which must adhere to one of the standards we support (OpenAPI/Swagger and AsyncAPI).
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/markdown-support.md:140]:
```suggestion
Markdown files can be included as an [external reference](/help/specification-support/references) within your API definition with the $ref syntax `$ref: "./path/to/local-markdown.md"`.
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (ChristopheDujarric) [inline src/_help/enhance-documentation-content/markdown-support.md:159]:
```suggestion
These files will be rendered within your documentation as if they were part of your API definition.
```

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [reply src/_help/enhance-documentation-content/markdown-support.md:8]: Works with the 2. `document` is OK here, but `definition` works better imo 👍

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:57]:
```suggestion
### Information call-outs
```

We need the same title level as for `Titles & headings`.

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:73]:
```suggestion
### Images
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [inline src/_help/enhance-documentation-content/markdown-support.md:82]:
```suggestion
#### Image sizing
```

File: src/_help/enhance-documentation-content/markdown-support.md
Comment (scharrier) [reply src/_help/enhance-documentation-content/markdown-support.md:140]: Both work here, but reading your comments Chris, I think that it does make sense to only refer to the "document" word when talking about the file. Here, we talke about the spec usage, so the definition word seems to work better 👍

**Review by scharrier — APPROVED** [review]
I left some final comments that need to be fixed before merging. I also validated most of Chris's proposal (document VS definition) 👍 

You won't need another review from myself, once everything is fixed this PR can be merged!

**Review by ChristopheDujarric — APPROVED** [review]

**Pull request merged**
